AllHitAnalyze.py

```
@@ -1,234 +1,234 @@
+				if idx < 0:
+					idx = int(idx)-1;
+				else:
+					idx = int(idx);
+
+				if (idx >= 0) and (idx < 250):hitMapB[idx] += 1;
+				elif (idx >= 250):hitMapB[250] += 1;
+				else:hitMapA[idx] += 1;
+
+		self.xAxis = [i for i in range(-150,251)];
+		self.yAxis = hitMapA + hitMapB;
+		self.avg = sum([ids[0]*ids[1]/self.sumYnum for ids in zip(self.xAxis,self.yAxis)]) if self.sumYnum > 0 else 0;
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)